packages/vscode-extension/src/handlers/createPluginWithManifestHandler.ts

```diff
@@ -35,7 +35,7 @@ export async function createPluginWithManifest(args?: any[]): Promise<Result<any
     TelemetryEvent.CreatePluginWithManifestStart,
     getTriggerFromProperty(args)
   );
-  if (!args || args.length < 2) {
+  if (!args || args.length < 2 || args.length > 3) {
```

Comment:
how about args.length !== 2

Reply:
It could be args.length !== 2 || args.length !==3, but I think the above one is easier to update if we need to add parameters

Reply:
Oh, I know~